components/Downloads/Release/BitnessDropdown.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import type { FC } from 'react';
+import { useEffect } from 'react';
+import semVer from 'semver';
+
+import Select from '@/components/Common/Select';
+import { useDetectOS } from '@/hooks/react-client';
+import { useReleaseContext } from '@/providers/releaseProvider';
+import { getBitnessItems } from '@/util/downloadUtils';
+
+const BitnessDropdown: FC = () => {
+  const { bitness: userBitness } = useDetectOS();
+  const {
+    state: { bitness, os, release },
+    dispatch: { setBitness },
+  } = useReleaseContext();
+
+  const hasWindowsArm64 = semVer.satisfies(release.version, '>= 19.9.0');
+
+  useEffect(() => {
+    setBitness(String(userBitness));
+  }, [setBitness, userBitness]);
+
+  return (
+    <Select
+      values={[
+        {
```

Comment:
This can be an inline object maybe? `{ items: getBitnessItems(os, hasWindowsArm64) }`